### PR TITLE
Update shipment.md

### PR DESCRIPTION
**Pull request by 7048730**
- add `recipient` according to upcoming shipper commit

### CODE REVIEW

File: docs/shipment.md [hunk 1]
```diff
@@ -3,6 +3,43 @@
...
 This section covers shipment related requests, including repository file management.
 
+
+### Packaging
+
+Currently, the shipment process always creates [BagIt](http://tools.ietf.org/html/draft-kunze-bagit) bags to package a compendium.
+
+### Supported recipients
+
+Use the _recipient_ endpoint to find out, which repositories are available and configured.
```
Comment (nuest) [inline docs/shipment.md]: Maybe add a description of the content:

> The reponse is list of tuples with `id` and `name` of repositories.
> `id` is the repository identifier to be used in requests to the `/shipment` endpoint, e.g. to define the recipient.
> `label` is a human-readable text string suitable for display in user interfaces.

File: docs/shipment.md [hunk 2]
```diff
@@ -3,6 +3,43 @@
...
+		"label": "Eudat b2share Sandbox"
+	}, {
+		"id": "zenodo",
+		"label": "Zenodo Sandbox"
+	}]
+}
+```
+
+This makes use of the following webservices:
+- `eudat` ([EUDAT b2share](https://b2share.eudat.eu/)).
```
Comment (nuest) [inline docs/shipment.md]: Inconsistent capitalization of EUDAT (vs. Eudate in line 27).

**Comment by 7048730** [comment]
added the requested changes

**Comment by nuest** [comment]
**Ready to be merged** as soon as the implementation is ready.

**Pull request merged**